index.html

```diff
@@ -4,360 +4,402 @@
+                feature or website Look through some of my work and experience!
+                If you like what you see and have a project you need coded,
+                don’t hestiate to contact me.
               </p>
             </div>
-            <div class="project-tag-container">
-              <ul class="tag-item">
-                <li class="tags"><a class="tag-btn" href="#">html</a></li>
-                <li class="tags"><a class="tag-btn" href="#">css</a></li>
-                <li class="tags"><a class="tag-btn" href="#">javaScript</a></li>
+            <section class="connect-section">
+              <h3 class="link-item">
+                <a href="#" class="connect">LET'S CONNECT</a>
+              </h3>
+            </section>
+            <div class="social">
+              <ul class="icon-svgs">
+                <li class="icon"><a class="svg twitter" href="#"></a></li>
+                <li class="icon"><a class="svg linkedlin" href="#"></a></li>
+                <li class="icon"><a class="svg medium" href="#"></a></li>
+                <li class="icon"><a class="svg git" href="#"></a></li>
+                <li class="icon"><a class="svg hand" href="#"></a></li>
               </ul>
             </div>
-            <div class="projects-btn">
-              <input type="button" value="See Project" class="project-button" />
-            </div>
-          </div>
-         </div>
-        </div>
-        <div class="content work-projects work-two">
-          <div class="item project-header">
-            <div class="projectsimage">
-              <img
-                src="assets/images/portfolio-image3.PNG"
-                alt="project image title"
-                class="headimage"
-              />
-            </div>
           </div>
-          <div class="cards-description">
-          <div class="item project-description">
-            <div class="project-title-container">
-              <h4 class="description-title">Tonic</h4>
-            </div>
-            <div class="project-subtitle-container">
-              <ul class="subtitle-items">
-                <li class="list-less"><p class="listinfo">CANOPY</p></li>
-                <li class="list"><p>Back End Dev</p></li>
-                <li class="list"><p>2015</p></li>
-              </ul>
-            </div>
-            <div class="project-article-container">
-              <p class="description-item description-item-b">
-                A daily selection of privately perosonalized reads; no accounts
-                or sign-ups required.
-              </p>
-            </div>
-            <div class="project-tag-container">
-              <ul class="tag-item">
-                <li class="tags"><a class="tag-btn" href="#">html</a></li>
-                <li class="tags"><a class="tag-btn" href="#">css</a></li>
-                <li class="tags"><a class="tag-btn" href="#">javaScript</a></li>
-              </ul>
+        </section>
+        <section id="myworks">
```

Comment:
- [ ] In each of this section's projects, kindly ensure the languages "HTML, Ruby on Rails, CSS" etc.  get aligned perfectly to the left with the rest of the elements.

![image](https://user-images.githubusercontent.com/57421823/159891972-17da2bd3-7547-497c-a4c5-ba2b65722686.png)

That way, these elements will be more consistent with the others, and will look like in the Figma 💪